### ISSUE TITLE
Quickstart Terminal fails because $HOME is not set

### ISSUE THREAD

**Issue by frehner** [docker/toolbox#64]
Background: I have been using Kitematic for awhile, decided to remove it and start fresh with Toolbox.

After installation, I try to run the Quickstart Terminal and get the error
```
rm: cannot remove '//.docker/machine/machines/default': No such host or network path
Looks like something went wrong... Press any key to continue
```

(I'm not sure if the following is helpful/relevent, but I'll add it just in case)

I run Toolbox's Kitematic, and it creates the VBox VM and can run the hello-world-nginx container just fine.

After, I try to use the Quickstart Terminal again and it makes more progress, shows the Docker Whale, but then gives me the error

```
C:\Program Files\Docker Toolbox\start.sh: line 51: cd: HOME not set
Looks like something went wrong... Press any key to continue
```

**Comment by jeffdm** [comment]
Thanks for the issue.

Seems like the $HOME env var isn't set on your Windows computer. Try:

- Open `System` via start menu
- Click `Advanced system settings`
- Click `Environment Variables`
- Add a HOME environment variable:

![screen shot 2015-08-12 at 9 58 20 am](https://cloud.githubusercontent.com/assets/251292/9230711/ba1ed07e-40d8-11e5-855b-e14f47fde338.png)

**Comment by thaJeztah** [comment]
@jeffdm is this an issue with the QuickStart terminal or with ‘docker machine`?

IIRC there was some discussion around the proper way of detecting the "home" dir; perhaps `%USERPROFILE%` should be used, but I don't remember the preferred way

ping @ahmetpbalkan perhaps you remember?

**Comment by frehner** [comment]
Yup, that fixed it. And just to be sure, I removed the VBox VM that Kitematic created and ran the script again, and was able to go through everything without problems.  

@thaJeztah not sure if this is related to what you're asking but... doing `docker-machine ls` is telling me that the docker-vm and kitematic machines don't exist.  If it's not related to this, then I'll open another issue.

**Comment by jeffdm** [comment]
@frehner good to hear. Out of curiosity, what do you get if you:

- Open `cmd.exe` and type `echo %USERPROFILE%` ?
- Open `Git Bash` and type `echo $HOME`?

**Comment by thaJeztah** [comment]
oh, see my previous ping was auto-~~correc~~mangled

ping @ahmetalpbalkan perhaps you remember? https://github.com/docker/toolbox/issues/64#issuecomment-130371444

**Comment by ahmetalpbalkan** [comment]
We've seen this happening in the past if Git Bash was already installed on the machine. Answers to Jeff's questions here would be helpful.

**Comment by frehner** [comment]
@jeffdm 
cmd.exe: C:\Users\afrehner
git bash: /c/Users/afrehner

@ahmetalpbalkan indeed, I already had git bash installed on my machine.

**Comment by Mordead** [comment]
Same issue for me

I have to put this :+1: 
export HOME="D:/Profiles/<myUser>"
in the start.sh script

@jeffdm 
cmd.exe: d:\Profiles\<myUser>
git bash: /v

As @frehner , I already had git bash installed on my machine.

**Comment by jeffdm** [comment]
@Mordead thanks! this is helpful. I think the best thing here is to change the git bash installation to be non-silent in the next version.

**Comment by jeffdm** [comment]
This should be fixed in 1.8.1c. Thank again!
